etc/kayobe/kolla/inventory/overcloud-services.j2

```diff
@@ -0,0 +1,423 @@
+# This inventory section provides a mapping of services to components.
```

Comment:
I'm not sure we need this piece - I think this relates to preventing placing cinder-volume services on servers in the storage group.  That is important if Kayobe has also deployed and configured the Ceph nodes.  In Alaska's case, the SoftIron nodes are truly external to Kayobe.  I think you can drop this

Reply:
@jovial thoughts?

Reply:
Looking at https://github.com/SKA-ScienceDataProcessor/alaska-kayobe-config/blob/c75c17e05643419a73020f2af1fc1dbbdd8071f3/etc/kayobe/kolla.yml#L190 I think we will need to this change. By default kolla-ansible will deploy cinder-volume to the storage group, but as we don't define a storage group, and we don't map another group to the storage group, this will mean that cinder-volume will not be deployed.

Reply:
Alternatively we map the controllers into the kolla ansible storage group via kolla_overcloud_inventory_top_level_group_map?